oio/directory/admin.py

```diff
@@ -59,10 +59,12 @@ def election_debug(self, params, **kwargs):
         return body
 
     @loc_params
-    def election_leave(self, params, **kwargs):
+    def election_leave(self, params, id=None, **kwargs):
```

Comment:
`id` is a reserved keyword. Besides it's not clear what this ID is. Is it a service ID?